act/headings.md

```diff
@@ -0,0 +1,137 @@
+# Headings
+
+## Rule Identifier
+
+gel-foundations-headings
+
+## Rule Description
+
+Headings provide a semantic and visual hierarchical structure to a document.
+
+This rule tests that pages have a heading structure meets the requirement of BBC GEL Headings.
```

Comment:
"structure meets the requirement **technical** of BBC GEL Headings." ?

Reply:
That doesn't make sense to me but that might be my lack of awareness of how different documents are referred to. Here I mean this [Headings](https://bbc.github.io/gel/foundations/headings/) document, but I am aware that there are others. If there is a specific way of referencing these pages then let me know so I can use the same language throughout.